### ISSUE TITLE
Warning! Error encountered during contract execution [Bad instruction] #2586

### ISSUE THREAD

**Issue by Migson51** [ethereum/mist#2598]
Two days ago, I wanted to buy some Status. However due to an error from their side the transaction failed. My ether however was send away to oblivian. Now, somebody told me that more people had the same problem. How can we get our deserved tokens or all of our ether (included the wasted fees) back. Is there something I can do to the contract because I believe that the contract itself is valid. Please let me know!

PS: Please don't close this case unless I've been given a valid solution. 

Kind regards,

Miguel

![status 0](https://user-images.githubusercontent.com/29603732/27450119-e8c8f51a-578a-11e7-9f9c-76388d43ec4f.JPG)
![status 2](https://user-images.githubusercontent.com/29603732/27450118-e8c74152-578a-11e7-8987-26dfd43a653b.JPG)

**Comment by shalbadr** [comment]
Someone besides the two us has to be concerned about this, I saw several transactions on etherscan showing people experiencing the same problem. I just want to know if the transactions will be returned later on or the transactions fixed and contract executed....

**Comment by Migson51** [comment]
Yeah me too.. this is unprofessional behavior. They cannot turn their backs to us and do nothing. Something has to be done about this.

**Comment by tcsiwula** [comment]
+2

**Comment by jbhav24** [comment]
Same

**Comment by shalbadr** [comment]
anyone get there ether back yet?

**Comment by ironox** [comment]
I have the same problem:

https://etherscan.io/tx/0x04820fe0adb6940c2073606f2f6eebed0bf01522893f107ebcae9d3d1c9ba189

sadder news is  I have two：

https://etherscan.io/tx/0xde91b3a40d2a3d40cfd44ccbd2ad55132e5d4a87067aa3c9a958fd40af0a046e

**Comment by davidevallicella** [comment]
https://etherscan.io/tx/0x5fd2bfeaa8d6b270d233f7d7dce98a394ca4d6649d78dc8c4cb08a077234037e

**Comment by evertonfraga** [comment]
So, clearly, this issue is not about Mist.

Feel free to camp here for a while. But eventually, it'll be automatically closed :)

**Comment by radumpopescu** [comment]
I also had this issue when trying to deposit KICK on Etherdelta:
https://etherscan.io/tx/0x9bc0717ae33f1ded9ba35f5ba76aa4993056fbadb15741542cb77e685e6cf86e
It happened twice :(
